plugins/woocommerce/client/legacy/css/woocommerce.scss

```diff
@@ -1741,6 +1650,102 @@ p.demo_store,
 	}
 }
 
+
+/**
+ * Buttons
+ */
+.woocommerce:where(body:not(.woocommerce-block-theme-has-button-styles)),
+:where(body:not(.woocommerce-block-theme-has-button-styles)) .woocommerce {
```

Comment:
FYI to ease the review: I hadn't made any changes to the styles below :point_down: , they appear in the diff because they were moved from another section of the document.